test-harness/tests/specs/unit/HyperClientTest.cfc

```diff
@@ -1149,7 +1149,7 @@ component extends="coldbox.system.testing.BaseTestCase" {
 				} );
 			});
 
-			describe( "tasks", function(){
+			xdescribe( "tasks", function(){
```

Comment:
Restore this test

Reply:
Please restore all of the tests in this PR which have been `x`ed

Reply:
Yikes! My bad!